analytics.go

```diff
@@ -20,14 +20,6 @@ const Version = "3.0.0"
 // Endpoint for the Segment API.
 const Endpoint = "https://api.segment.io"
 
-// DefaultContext of message batches.
-var DefaultContext = map[string]interface{}{
-	"library": map[string]interface{}{
-		"name":    "analytics-go",
-		"version": Version,
-	},
-}
```

Comment:
I believe the reason this was exported so that users can attach their own default context to all the calls.